client/src/org/compiere/grid/ed/VLocator.java

```diff
@@ -625,7 +625,10 @@ private void setDefault_Locator_ID()
 			return;
 		}
 		MWarehouse wh = MWarehouse.get(Env.getCtx(), M_Warehouse_ID);
-		if (wh == null || wh.get_ID() <= 0) {

```

Comment:
Have you a similar change for WLocator? Please this change should be applied also for ZK UI

Reply:
Hi @yamelsenih,
No. The problem targetes only to swing.